config/src/main/java/com/alibaba/nacos/config/server/controller/ConfigServletInner.java

```diff
@@ -79,10 +83,18 @@ public ConfigServletInner(LongPollingService longPollingService, PersistService
      */
     public String doPollingConfig(HttpServletRequest request, HttpServletResponse response,
             Map<String, String> clientMd5Map, int probeRequestSize) throws IOException {
+        return doPollingConfig(request, response, clientMd5Map, probeRequestSize, false);
```

Comment:
long polling API no need to reactor to `v2` beacuse `v2` use grpc to listener config.